tests/reftests/git.test

```diff
@@ -0,0 +1,72 @@
+N0REP0
+### : Check the bevahiour when presented with a broken submodule
+### mkdir submodule
+### git -C ./submodule init -q
+### git -C ./submodule config core.autocrlf false
+### touch ./submodule/some-file
+### git -C ./submodule add ./some-file
+### git -C ./submodule commit -qm "first commit"
+### mkdir use-submodule
+### git -C ./use-submodule init -q
+### git -C ./use-submodule config core.autocrlf false
+### ### This test may fail locally depending on your git version
+### ### File protocol is removed locally since git 2.38,
+### ### to fix a CVE: https://www.cve.org/CVERecord?id=CVE-2022-39253
```

Comment:
I added that note

Reply:
That note isn't quite true. Only a couple of versions have file protocol disabled. More recent git versions work just fine

Reply:
Ah nevermind, what i'm saying is not true. My memory failed me.